packages/core-mobile/app/services/earn/utils.ts

```diff
@@ -262,9 +262,23 @@ export const getRandomValidator = (
     // get the first item in the array sorted by end time
     return validators.at(0) as NodeValidator
   }
-  const endIndex = validators.length >= 5 ? 4 : validators.length - 1
+
+  // get the validators with lowest delegation fee
+  const validatorsWithLowestFee = validators.filter(
+    validator => validator.delegationFee === validators[0]?.delegationFee
```

Comment:
Can you please update docs of getRandomValidator function parameter "validators", to something like "input sorted by delegationFee to take random item form"?
At first glance it's not obvious why comparing with first item.